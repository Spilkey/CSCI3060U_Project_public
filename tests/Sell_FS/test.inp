login
user02
sell
airpods meetups
100.00
26
logout
exit