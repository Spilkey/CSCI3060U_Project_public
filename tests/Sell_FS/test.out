03_airpods_meetup____________user02________026_100.00
00_user02__________FS_000000.00